pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -38,40 +53,33 @@ stdenv.mkDerivation (finalAttrs: {
     s
     servant
     shell-split-string
-  ]);
+  ];
 
   strictDeps = true;
 
-  buildPhase = ''
-    runHook preBuild
-
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
-
-    runHook postBuild
+  postPatch = ''
+    lispdir=$out/share/emacs/site-lisp/elpa/cask-${version} \
+      substituteAllInPlace bin/cask
   '';
 
-  installPhase = ''
-    runHook preInstall
-
-    mkdir -p $out/bin
-    install -Dm444 -t $out/share/emacs/site-lisp/cask *.el *.elc
-    install -Dm555 -t $out/share/emacs/site-lisp/cask/bin bin/cask
-    ln -s $out/share/emacs/site-lisp/cask/bin/cask $out/bin/cask
-
-    runHook postInstall
+  postInstall = ''
+    install -D -t $out/bin bin/cask
   '';
 
-  meta = with lib; {
+  passthru.updateScript = gitUpdater {
```

Comment:
Upstream changes `bin/cask` slightly in the latest version so our patch needs to be changed as well.

r-ryantm won't create an update PR for this package because it fails to build the latest version.  So we have to bump it manually.